omaha/common/config_manager.h

```diff
@@ -460,6 +460,11 @@ class ConfigManager {
   // build flavor or other configuration parameters.
   bool AlwaysAllowCrashUploads() const;
 
+  // Returns true if Authenticode signatures of update binaries should always be
+  // verified. If this is false (the default), then signatures are only verified
+  // in code red scenarios and in app commands.
+  bool AlwaysVerifyAuthenticodeSignatures() const;
```

Comment:
Similar to what I said before, it would be help if the naming is more specific.

Reply:
Fixed in 9477eacac348e46b2d643ef7e2e843a48947293c.